lection5/task3.js

```diff
@@ -0,0 +1,42 @@
+// 1*. Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом), 
+//   и выводить в консоль количество гласных и согласных букв в этом слове. 
+//   Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+
+const str = 'Kalabanga'; //слово
+const vowelsStr = 'aeiouy'; //гласные
+let vowels = 0;
+let consonants = 0;
+
+for (let char of str) {
+    if(vowelsStr.indexOf(char) > -1) {
+        vowels++
+      } else {
+        consonants++
```

Comment:
Word can contain "-" symbol. So need to have consonants string too and check if value is inclued in consonants

Reply:
done